src/rgw/rgw_sal_motr.cc

```diff
@@ -2546,6 +2694,12 @@ int MotrAtomicWriter::complete(size_t accounted_size, const std::string& etag,
                     << " etag: " << etag << " user_data=" << user_data << dendl;
   if (user_data)
     ent.meta.user_data = *user_data;
+
+  if (!obj.get_key().have_instance()) {
+    // generate-version-id_null, overwrite instance field.
```

Comment:
> generate-version-id_null

we decided against _null approach. can you update comment also here

Reply:
yes, done